NL/2DCode/raw/119.json

```diff
@@ -0,0 +1,52 @@
+{
+    "JSON": {
+        "ver": "1.0.0",
+        "nam": {
+            "fn": "van der Achternaam-Leeuwarden",
+            "fnt": "VAN<DER<ACHTERNAAMLEEUWARDEN",
+            "gn": "Voornaam",
+            "gnt": "VOORNAAM"
+        },
+        "dob": "2000-02-29",
+        "v": [
+            {
+                "tg": "840539006",
+                "vp": "",
+                "mp": "",
+                "ma": "",
+                "dn": 4,
+                "sd": 4,
+                "dt": "2021-02-18",
+                "co": "SD",
+                "is": "Ministry of Health Welfare and Sport",
```

Comment:
Again, syntactically correct - but I wouldn't expect MinVWS to be the issuer for Sudan...
is: this should really be a constrained valueset, but very few Member States have actually agreed what is going to be put there, at least at an EU level

Reply:
What if someone got the vaccination in SD, had their NL doctor confirm the vaccination, and NL doctor gives out the DGC, should there be SD here or NL?